src/validations/mta/mtaDiagnostic.ts

```diff
@@ -71,3 +71,11 @@ export function getSeverity(
     ? DiagnosticSeverity.Warning
     : DiagnosticSeverity.Error;
 }
+
+export async function getValidation(
+  modulePath: string,
+  devMtaExts: string[] | undefined
+): Promise<Record<string, mta.Issue[]>> {
+  const mta = new Mta(modulePath, false, devMtaExts); // temp file is not relevant in our scenario
+  return await mta.validate();
```

Comment:
The `await` is unnecessary here since you return a promise anyway (you can just use `return mta.validate();`)